docs/staging/dmb/dmb-aspects-of-a-good-application.md

```diff
@@ -0,0 +1,118 @@
+
+Example:
+
+* Evelyn is a member of the Canonical Foundations Toolchain Squad. They have worked extensively on Zig, not just on the toolchain but on many leaf packages. Over time, Evelyn has built up a portfolio of helping across Ubuntu, not just with Zig but in other areas. First, they applied for a speciality packageset `zig-packages`, and got endorsements from current packageset uploaders and a Core Dev, all within Canonical Foundations (but including a Core Dev outside their immediate team). When applying for Core Dev, they went through their uploads and asked sponsors from Debcrafters and Server to provide endorsements as well. This shows a work across immediate Canonical Teams.
+
+* Frederick is a member of Canonical Support Engineering. Their work has been focused on SRUs of fixes to stable releases. They attained SRU-dev and are working toward Core Dev. In their first application, they only included endorsements from current Support Engineering members. This application will raise concerns from the DMB over Frederick’s communication and culture fit.
+
+* Günther is working in the project for quite some time. They have 2 contributions to 3 packages each - and the people sponsoring those have endorsed Günther. But those have been long ago and there are two other sponsors that have done the majority of the work of Günther in recent years. Without an explanation that will make people question if that recent work was not good quality.
+
+* John wants to apply for the rather new debcrafter package set, but it only has one member so far. This is not a dead-lock due to requiring endorsements from the same level - endorsements by people sponsoring for him into other similar package sets or core developers would also back his case
+
+### Partial endoresement
+
+In the same context it is important to understand that people can partially endorse you.
+To be clear, please do not provide us with a puzzle of 27 fragmented partial endoresements - but a few can be helpful to fill some gaps.
+If you have contacts that worked with you in depth, but only on particular aspects you should still consider asking them for endorsement.
+
+Yet again it is important to avoid misunderstandings.
+They should clearly state if a) they have doubts in the non-mentioned areas or b) just do not know all aspects, but are happy to endorse you for those they worked with you.
+
+Example:
+
+* Paulina applies, has two full endoresments but also has worked with many more project members. 3 of them provide partial endorsements, one speaking about the kind interaction on bugs and the great chat behavior - another about great things they have sponsored - and another about a very complex transition you helped them. All of them mention that they do not cover the other aspects of a full endorsement because they just didn't have this kind of exposure to you yet.
```

Comment:
```suggestion
* Paulina applies, has two full endorsements but also has worked with many more project members. Three of them provide partial endorsements: one speaking about the kind interaction on bugs and the great chat behavior; another about great things they have sponsored; and another about a very complex transition they helped with. All of them mention that they do not cover the other aspects of a full endorsement because they just didn't have this kind of exposure to Paulina yet.
```